### PR TITLE
Feature/show variables

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/components/EventsTimeline/TimelineEventItem.tsx [hunk 1]
```diff
@@ -18,13 +19,20 @@ type Props = {
...
+type ItemType = {
+  label: string
+  value: string | boolean
+  value_verbose: string
+}
+
 const TimelineEventItem: React.FC<Props> = ({ fields, caseEvents, title = "", dateField = "date_created", pathName, isOpen = false, useTransparentBackground = false, itemCount = 0 }) => {
 
   // This situation would be considered a problem within the data returned from the API
   if (caseEvents.length === 0) return null
```
Comment (gilleswittenberg) [inline src/components/EventsTimeline/TimelineEventItem.tsx:31]: Think you probably want to move line 35 above here. And check `if (caseEvents.length === 0 && items.length === 0)`.

File: src/components/EventsTimeline/TimelineEventItem.tsx [hunk 2]
```diff
@@ -44,10 +52,25 @@ const TimelineEventItem: React.FC<Props> = ({ fields, caseEvents, title = "", da
...
+            <>
+              <EventWrapper fields={ fields } caseEvent={ caseEvent } />
+              { items.length > 0 &&
+                <Dl>
+                  { items.map(({ label, value, value_verbose }, index) => { //value_verbose is the humanreadable variant of value
+                    if(value === false) return null //hide unchecked checkboxes
+                    return (
+                      <div key={ `${ label }_${ index }` }>
+                        <dt>{ label }</dt>
+                        <dd>{ typeof value === "string" && value_verbose }</dd>
```
Comment (gilleswittenberg) [inline src/components/EventsTimeline/TimelineEventItem.tsx:64]: This will be empty when value is not a string?

**Review by gilleswittenberg — APPROVED** [review]

File: src/components/EventsTimeline/TimelineEventItem.tsx (hunk 2 above)
Comment (gilleswittenberg) [reply src/components/EventsTimeline/TimelineEventItem.tsx:64]: And checks for `value`, but prints `value_verbose`. Is that correct? Maybe do: `typeof value_verbose === "string" ? value_verbose : "-"`

File: src/components/EventsTimeline/TimelineEventItem.tsx (hunk 1 above)
Comment (remcohoff) [reply src/components/EventsTimeline/TimelineEventItem.tsx:31]: Not really. `Items` is only used for the fields in `variables`. Not every `caseEvent` has variables with fields in it but still has to be shown (for instance every static form)

File: src/components/EventsTimeline/TimelineEventItem.tsx (hunk 2 above)
Comment (remcohoff) [reply src/components/EventsTimeline/TimelineEventItem.tsx:64]: Yes. It's valid html

**Pull request merged**